web/libs/ui/src/lib/popover/popover.tsx

```diff
@@ -0,0 +1,41 @@
+import type * as React from "react";
+import {
+  Popover as ShadPopover,
+  PopoverTrigger,
+  PopoverContent,
+  PopoverAnchor,
+} from "../../shad/components/ui/popover";
+import { cn } from "@humansignal/shad/utils";
+
+export interface PopoverProps extends React.ComponentProps<typeof ShadPopover> {
+  trigger: React.ReactNode;
+  children: React.ReactNode;
+  align?: "start" | "center" | "end";
+  sideOffset?: number;
+  className?: string;
+}
+
+export function Popover({
+  trigger,
+  children,
+  align = "center",
+  sideOffset = 4,
+  className,
+  ...props
+}: PopoverProps) {
+  return (
+    <ShadPopover {...props}>
+      <PopoverTrigger data-slot="popover-trigger" asChild>{trigger}</PopoverTrigger>
+      <PopoverContent align={align} sideOffset={sideOffset} 
+        data-slot="popover-content"
+        className={cn(
```

Comment:
We should use a combination of twMerge(clsx()) so, if passed tailwind classes, they are resolved and overriden when neccessary

Reply:
good call, will do